front-end/src/components/Layout.jsx

```diff
@@ -0,0 +1,34 @@
+import PropTypes from 'prop-types'
+
+export const Layout = ({ children }) => {
+
+
+  return (
+    <div className="meal-body">
+      <div className="meal-header">
+        <div className="meal-header__left">
+          <div className="meal-header__logo">
+            <svg
```

Comment:
It is better to create a new  logo component from the SVG  part and then import, therefore you code will be cleaner and the icon could be reuse again at other places